integration/test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -93,6 +93,53 @@ func TestWriteLogsToCloudWatch(t *testing.T) {
 	}
 }
 
+// Validate https://github.com/aws/amazon-cloudwatch-agent/issues/447
+func TestRotatingLogsDoesNotSkipLines(t *testing.T) {
```

Comment:
Should we split this out into a new dir so it can run at the same time or is it fine being apart of the same test?

Reply:
Personally I think it's fine being in here, but I'll defer to you if you think it should be separated into it's own dir and executed concurrently. I feel like this is pretty small in the grand scheme of our testing strategy.

Reply:
IDC you wrote the code, your choice.

Reply:
I'll leave it here then cause it's easier on me